dash_bio/component_factory/_clustergram.py

```diff
@@ -61,77 +61,77 @@ def Clustergram(
+    :param (function) link_fun: Function to compute the linkage matrix from the
                                pairwise distances
                                (see scipy.cluster.hierarchy.linkage)
-    :param (dict) colorThreshold: Maximum linkage value for which unique
+    :param (dict) color_threshold: Maximum linkage value for which unique
                                   colors are assigned to clusters; 'row' for
                                   rows, and 'col' for columns (default 0)
-    :param (bool) optimalLeafOrder: Enable or disable calculation to determine
+    :param (bool) optimal_leaf_order: Enable or disable calculation to determine
                                     leaf order that maximizes the similarity
                                     between neighbouring leaves
-    :param (list) colorMap: Optional colorscale for dendrogram tree
-    :param (dict) colorList: Optional list of colors to use for row and col
+    :param (list) color_map: Optional colorscale for dendrogram tree
+    :param (dict) color_list: Optional list of colors to use for row and col
                              dendrograms
-    :param (double) displayRange: Standardized values from the dataset that
+    :param (double) display_range: Standardized values from the dataset that
                                   are below the negative of this value will
                                   be colored with one shade, and the values
                                   that are above this value will be colored
                                   with another in the heatmap (3 by default)
-    :param (bool) symmetricValue: Forces the colorscale of the heatmap to be
+    :param (bool) symmetric_value: Forces the colorscale of the heatmap to be
                                   centered around zero (True by default)
-    :param (bool) logTransform: Transforms the data to a logarithmic axis
+    :param (bool) log_transform: Transforms the data to a logarithmic axis
                                 with a basis of two (False by default)
-    :param (list/float) displayRatio: The dendrograms' heights with respect to
+    :param (list/float) display_ratio: The dendrograms' heights with respect to
                                      the size of the heatmap; with one element,
                                      both the row and column dendrograms have
                                      the same ratio; with two, the row
                                      dendrogram ratio corresponds to the first
                                      element of the list (default: 0.2)
-    :param (function) imputeFunction: The function used to impute missing data.
+    :param (function) impute_function: The function used to impute missing data.
                                       It should take as input the dataset and
                                       output a dataset with imputed values.
```

Comment:
I'm not sure this description is clear enough... As is, it sounds like there's not necessarily any missing data in the original dataset, but we're imputing some... Instead, I believe that we're talking about the *representation* of missing data here. Could it be: "Function used to set the representation of missing values. It takes the whole dataset as input and outputs it with missing values converted into the desired representation."?

Reply:
Ok, it's not actually a function. Will re-review after update.